developer-docs-site/docs/tutorials/your-first-nft.md

```diff
@@ -8,351 +8,483 @@ import TabItem from '@theme/TabItem';
 import ThemedImage from '@theme/ThemedImage';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-:::caution
+# Your First NFT
 
-The following tutorial is a work in progress. The Aptos (Non-Fungible) Token specification has not been formalized.
+This tutorial describes, in the following step-by-step approach, how to create and transfer NFTs on the Aptos Blockchain. The Aptos implementation for core NFTs or Tokens can be found in [token.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move).
 
-:::
+## Step 1: Pick an SDK
 
-# Your First NFT
+* [Official Aptos Typescript SDK][typescript-sdk]
+* [Official Aptos Python SDK][python-sdk]
+* Official Aptos Rust SDK -- TBA
+
+## Step 2: Run the Example
 
-An [NFT](https://en.wikipedia.org/wiki/Non-fungible_token) is a non-fungible token or data stored on a blockchain that uniquely defines ownership of an asset. NFTs were first defined in [EIP-721](https://eips.ethereum.org/EIPS/eip-721) and later expanded upon in [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155). NFTs typically comprise of the following aspects:
+Each SDK provides an examples directory. This tutorial covers the `simple-nft` example.
 
-- A name, the name of the asset, which must be unique within a collection
-- A description, the description of the asset
-- A URL, a non-descript pointer off-chain to more information about the asset could be media such as an image or video or more metadata
-- A supply, the total number of units of this NFT, many NFTs have only a single supply while those that have more than one are referred to as editions
+Clone `aptos-core`:
+```sh
+git clone git@github.com:aptos-labs/aptos-core.git ~/aptos-core
+```
+
+<Tabs groupId="sdk-examples">
+  <TabItem value="typescript" label="Typescript">
 
-Additionally, most NFTs are part of a collection or a set of NFTs with a common attribute, e.g., theme, creator, or minimally contract. Each collection has a similar set of attributes:
+  Navigate to the Typescript SDK examples directory:
+  ```sh
+  cd ~/aptos-core/ecosystem/typescript/sdk/examples/typescript
+  ```
 
-- A name, the name of the collection, which must be unique within the creator's account
-- A description, the description of the asset
-- A URL, a non-descript pointer off-chain to more information about the asset could be media such as an image or video or more metadata
+  Install the necessary dependencies:
+  ```
+  yarn install
+  ```
+
+  Run the `simple_nft` example:
+  ```sh
+  yarn run simple_nft
+  ```
+  </TabItem>
+  <TabItem value="python" label="Python">
+
+  Navigate to the Python SDK directory:
+  ```sh
+  cd ~/aptos-core/ecosystem/python/sdk
+  ```
+
+  Install the necessary dependencies:
+  ```
+  curl -sSL https://install.python-poetry.org | python3
+  poetry update
+  ```
+
+  Run the `transfer-coin` example:
+  ```sh
+  poetry run python -m examples.simple-nft
```

Comment:
is this necessary? 🙄

Reply:
Yes, we want to make the tutorial stand alone.